app/merginapi.h

```diff
@@ -47,6 +47,11 @@ struct MerginProject
   QList<int> writers;
 };
 
+struct ProjectDiff
+{
+  QHash<QString, QList<MerginFile>> changes;
```

Comment:
For ProjectDiff structure I was expecting that added, modified, removed, (renamed) files would be individual member variables so that we do not need a QHash...